template/src/utilities/utils.ts

```diff
@@ -39,3 +39,7 @@ export function validateEmail(email: string): boolean {
 export function isDarkMode(): boolean {
   return Appearance.getColorScheme() === 'dark'
 }
+
+export function isObject<T>(val: T): boolean {
+  return typeof val === 'object' && val?.constructor !== FormData && val !== null
```

Comment:
please check error Type "Property 'constructor' does not exist on type 'NonNullable<T>'"